cmd/persistenceCore/cmd/root.go

```diff
@@ -255,6 +297,8 @@ func (ac appCreator) newApp(
 		baseapp.SetSnapshotStore(snapshotStore),
 		baseapp.SetSnapshotInterval(cast.ToUint64(appOpts.Get(server.FlagStateSyncSnapshotInterval))),
 		baseapp.SetSnapshotKeepRecent(cast.ToUint32(appOpts.Get(server.FlagStateSyncSnapshotKeepRecent))),
+		baseapp.SetIAVLCacheSize(cast.ToInt(appOpts.Get(server.FlagIAVLCacheSize))),
```

Comment:
this flag is not set in start.go of cosmos-sdk v0.45.9, wonder if this statement works at all..

Reply:
It worked for evmos, also this value can be set in app.toml

Reply:
yes, it can be picked from app.toml.
might have to add it to instructions then
@Anmol1696

Reply:
noted